stonesoup/sampler/base.py

```diff
@@ -0,0 +1,5 @@
+from ..base import Base
+
+
+class Sampler(Base):
+    """Sampler base class"""
```

Comment:
Should give a description of what a general sampler does.

Reply:
Description added